24_find_lab/findlab.txt

```diff
@@ -1,24 +1,72 @@
 1. Write command lines to find using "find":
 
-  -. All files owned by yourself in /tmp folder
+  
+-. All files owned by yourself in /tmp folder
 
-  -. All files and directories not owned by you in /tmp folder
+  
+I typed find /tmp/ -user yulia and mostly receive permission messages
+
+
+-. All files and directories not owned by you in /tmp folder
 
-  -. All files, folders and links in your home folder whose name ends with a ".c"
+ 
+
+I typed find /tmp/ -user root and mostly receive permission messages, same as with find /tmp/ -group root 

```

Comment:
See previous comment as for suppressing the errors. Moreover some files may belong to users who are not you neither root... Better to use `find /tmp/ -not -user $USER`